recipe/meta.yaml

```
@@ -1,6 +1,6 @@
 {% set version = "4.12.0" %}
-{% set constructor_version = "3.2.2" %}
-{% set python_version = "3.9.9" %}
+{% set constructor_version = "3.3.1" %}
+{% set python_version = "3.9.13" %}
```

```suggestion
{% set python_version = "3.9.15" %}
```

Available as of today!

---

Man, those minor versions get released fast!